lib/app/custom_widgets/header_with_league_and_timer/elements/match_in_progress_timer.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:score_live/presentation/constants/app_colors.dart';
+import 'package:score_live/presentation/constants/app_const_variables.dart';
+import 'package:score_live/presentation/constants/text_styles.dart';
+
+class MatchInProgressTimer extends StatelessWidget {
+  const MatchInProgressTimer({
+    super.key,
+    required this.matchStatusShort,
+    required this.matchTimeElapsed,
+  });
+
+  final String matchStatusShort;
+  final int matchTimeElapsed;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: 30,
+      width: 50,
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(24),
+        color: matchStatusShort == AppConstVariables.matchTimeToBeDefined ||
```

Comment:
You are using this condition three times. Please export it to a separate variable.

Reply:
fixed